app/controllers/hq/orphans_controller.rb

```diff
@@ -3,4 +3,63 @@ class Hq::OrphansController < HqController
   def show
     @orphan = Orphan.find(params[:id])
   end
+
+  def index
+    @orphans = Orphan.paginate(:page => params[:page])
+  end
+
+  def create
+    @orphan ||= Orphan.new
```

Comment:
Update: It just dawned on me that new orphan records are never created via the orphans controller but rather via the orphan_lists controller. I've submitted PR #351 to fix the admin/orphans controller.

@christophermlne can you please remove the `create` action and update specs accordingly?